Documentation/configuration.md

```diff
@@ -3,7 +3,7 @@
 ## Automatic
 
 You can use an USB drive with HassOS to configure network options, SSH access to the host and to install updates.
-Format a USB stick with FAT32/EXT4/NTFS and name it `CONFIG`. Use the following directory structure within the USB drive:
+Format a USB stick with FAT32/EXT4/NTFS and name it `CONFIG`. Alternative you can create a `CONFIG` folder inside boot partition. Use the following directory structure within the USB drive:
```

Comment:
I am not sure what boot partition is. 

> 1. Put the SD card in your SD card reader.
> 2. Open Etcher, select the Hass.io image and flash it to the SD card.
> 3. WiFi and Static IP setup only: Format a USB-Stick with name CONFIG, create a folder named network and within that folder a file named my-network. Copy one of the examples to the my-network file.

For step3, is it correct to create a folder in my SD card "/boot/CONFIG" before I install hassos to the Pi?